src/Platform.Handlers/src/Xamarin.Platform.Handlers/Platform/Android/ButtonExtensions.Android.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
```

Comment:
with the new MultiTarget.targets, do you really need the `Android` folder AND the `.Android.cs` suffix ?

Reply:
No, I removed the suffix on these files